build-configs/obdach/index.ts

```diff
@@ -0,0 +1,109 @@
+import { OBDACH_ASSETS } from '../AssetsType'
+import {
+  AndroidBuildConfigType,
+  CommonBuildConfigType,
+  iOSBuildConfigType,
+  WebBuildConfigType
+} from '../BuildConfigType'
+import { APP_STORE_TEAM_ID } from '../common/constants'
+import mainImprint from './mainImprint'
+import { lightTheme } from './theme'
+
+const APPLE_APP_ID = '1072353915'
+
+const commonObdachBuildConfig: CommonBuildConfigType = {
+  appName: 'Vernetztes Obdach',
+  appIcon: 'app_icon_obdach',
+  lightTheme,
+  assets: OBDACH_ASSETS,
+  cmsUrl: 'https://admin.vernetztesobdach.de',
+  hostName: 'vernetztesobdach.de',
+  allowedHostNames: ['admin.vernetztesobdach.de'],
+  internalLinksHijackPattern:
+    'https?:\\/\\/((admin\\.)?vernetztesobdach\\.de)(?!\\/(media|[^/]*\\/(wp-content|wp-admin|wp-json))\\/.*).*',
+  featureFlags: {
+    floss: false,
+    pois: true,
+    newsStream: true,
+    pushNotifications: true,
+    introSlides: true,
+    jpalTracking: false,
+    sentry: false,
+    developerFriendly: false,
+    fixedCity: null,
+    cityNotCooperatingTemplate: null
+  },
+  aboutUrls: {
+    default: 'https://vernetztesobdach.de/about/',
+    en: 'https://vernetztesobdach.de/en/about/'
+  },
+  privacyUrls: {
+    default: 'https://vernetztesobdach.de/datenschutz/',
+    en: 'https://vernetztesobdach.de/en/privacy/'
+  }
+}
+export const androidObdachBuildConfig: AndroidBuildConfigType = {
+  ...commonObdachBuildConfig,
+  splashScreen: true,
+  applicationId: 'tuerantuer.app.integreat',
+  sha256CertFingerprint:
+    '66:2E:43:DC:9C:75:DE:7D:99:C5:BA:A9:19:DA:0F:BB:5E:6C:13:3D:03:E8:1D:FB:EF:87:F5:4B:F1:80:D4:B1',
+  googleServices: {
```

Comment:
Remove google services (for iOS as well), only needed for push notifications on native

Reply:
i meant set it to `null`, you don't need to adjust the types for that.